bin/server.py

```diff
@@ -145,13 +147,21 @@ async def change_root_password(self, request, password):
 
     @only_if_initialized
     @remote
-    async def sync_configuration(self, request, image_configuration_params):
-        request.ret(READY)
+    async def get_configuration_main_params(self, request):
```

Comment:
Implement only two remote procedures: `get_configuration` and `set_configuration`.